src/script/script.cpp

```diff
@@ -273,6 +273,41 @@ bool CScript::IsBribe() const
     return (this->Find(OP_BRIBE));
 }
 
+bool CScript::IsMTHashCommit() const
+{
+    size_t size = this->size();
+    if (size != 39) // sha256 hash + opcodes
+        return false;
+
+    if ((*this)[0] != OP_RETURN ||
+            (*this)[1] != 0x44 || // D
```

Comment:
They don't mean anything. I did DRVMT for the drivechain MT hash commits, and DRVWT^ for drivechain WT^ commits. I arbitrarily chose what looked good to me, it just has to be a unique enough identifier for those scripts. Totally open to doing it another way though.